R/forecast.R

```diff
@@ -65,6 +65,9 @@ as_forecast <- function(data,
 #' @param sample_id (optional) Name of the column in `data` that contains the
 #'   sample id. This column will be renamed to "sample_id". Only applicable to
 #'   sample-based forecasts.
+#' @param check_forecast_type Logical, defaults to `FALSE`. Should the
+#' `forecast_type` be checked against the type implied by the `data`?
```

Comment:
```suggestion
#' @param check_forecast_type Logical, defaults to `FALSE`. Should the
#' `forecast_type` be checked against the type implied by the `data`? If
#' `forecast_type` is NULL it will be inferred from the data before checking.
```